k8s/base/quotas.json

```diff
@@ -1,8 +1,13 @@
 {
+":requests.cpu":                  { "base": 2, "coefficient": 0 },
```

Comment:
You could also eliminate the "base" and "coefficient in all of these as they are used for the quota multiplier.

Reply:
Will do in a follow-up patch!

Reply:
Opened #80 for it.